lib/RegisterAll.cpp

```diff
@@ -28,6 +30,8 @@ void mlir::tt::registerAllDialects(mlir::DialectRegistry &registry) {
               mlir::scf::SCFDialect, mlir::cf::ControlFlowDialect,
               mlir::tosa::TosaDialect, mlir::vector::VectorDialect,
               mlir::emitc::EmitCDialect>();
+
+  mlir::stablehlo::registerAllDialects(registry);
```

Comment:
Can you try a clean build + `rm -rf third_party/stablehlo` + `TTMLIR_ENABLE_STABLEHLO=OFF`? It seems like this and the include above needs to be protected with a #define guard

Reply:
I think this and the include above need to be protected by the #define